apps/system/utils/security_level_cmd.c

```diff
@@ -0,0 +1,97 @@
+/****************************************************************************
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/ss_info.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <apps/shell/tash.h>
+#include <security/security_ss.h>
+
+/****************************************************************************
+ * Preprocessor Definitions
+ ****************************************************************************/
+
+/********************************************************************************
+ * Private Function Prototypes
+ ********************************************************************************/
+
+static void tash_set_security_level(int argc, char **args)
+{
+	int security_level;
+	char ss_path[7] = { 0, };
+        security_handle hnd;
+	security_data ss_data = {NULL, 0};
+	security_error ret;
+
+	/* Change security_level according to user input */
+	ss_data.length = 2;
+	if (!strncmp(args[1], "high", 5)) {
+		ss_data.data = "1";
+		security_level = 1;
+	} else if (!strncmp(args[1], "low", 4)) {
+		ss_data.data = "0";
+		security_level = 0;
+	} else {
+		printf("Invalid argument\n");
```

Comment:
Please reference other commands. All commands provide the `help` option to show how to use. User of the command could be a non-developer so that Kconfig is not enough to check how to use. Even he or she is a developer, he or she could miss the Kconfig description.

Reply:
I add --help option in set_sec_level. Thank you for your review.